arangod/GeneralServer/GeneralServerFeature.cpp

```diff
@@ -175,6 +177,14 @@ void GeneralServerFeature::collectOptions(std::shared_ptr<ProgramOptions> option
                      "trusted origin URLs for CORS requests with credentials",
                      new VectorParameter<StringParameter>(&_accessControlAllowOrigins));
 
+  options->addOption("--http.redirect-root-to",
+                    "redirect of root URL",
+                    new StringParameter(&_redirectRootTo));
```

Comment:
This new option, as well as the following one, should get a `.setIntroducedIn(30712)` extension.